content/en/docs/v3.4/integrations.md

```diff
@@ -100,6 +100,10 @@ description: A listing of etcd tools and client libraries
 - [marshall-lee/etcd.erl](https://github.com/marshall-lee/etcd.erl) - Supports v2
 - [zhongwencool/eetcd](https://github.com/zhongwencool/eetcd) - Supports v3+ (GRPC only)
 
+### Elixir
+
+- [team-telnyx/etcdex](https://github.com/team-telnyx/etcdex) - Supports v3+ (GRPC only)
```

Comment:
```suggestion
- [team-telnyx/etcdex](https://github.com/team-telnyx/etcdex) - Supports v3 (GRPC only)
```

Reply:
Please check eetcd just above.

Reply:
It isn't a big deal although it would be better to use uniform naming, and I already approved the PR.